demo/meeting/package.json

```diff
@@ -26,7 +26,8 @@
     "start:client": "npm run build:fast && webpack-dev-server",
     "start:backend": "node server.js",
     "start:fast": "concurrently \"npm run start:client\" \"npm run start:backend\"",
-    "start": "npm run deps && yalc add amazon-chime-sdk-component-library-react && npm install && npm run start:fast"
+    "start": "npm run deps && yalc add amazon-chime-sdk-component-library-react && npm install && npm run start:fast",
+    "start:no-yalc": "npm run deps && npm install && npm run start:fast"
```

Comment:
hmm we should not need to run deps in this case as it just does yalc publish. 
And can we have better name (like start:npm or start:release) instead of start:no-yalc?
Also need to do a build:release command too.

Reply:
The build happens in the start:fast script, so we dont need to. But I'll update the script names.

Reply:
Would it be better to have a script with `yalc` separately and not edit the existing script triggers? If existing scripts are changed we need to surface that as we may run into any errors as I did yesterday.

Reply:
I'm not changing any existing scripts, just adding start:release and deps:release, which dont use yalc

Reply:
We do have a build command though. What happen if I just want to check whether it build or not?

Reply:
I think normally we should be defaulting to use the yalc generated module, but I can see a scenario where we're not doing a release where we might want to build a different version of the react library into the meeting demo. Ive added `build:release' so that we can skip the yalc step